src/Illuminate/Database/Query/Builder.php

```diff
@@ -1613,7 +1613,7 @@ public function sum($column)
 	 * Retrieve the average of the values of a given column.
 	 *
 	 * @param  string  $column
-	 * @return mixed
+	 * @return float|int
```

Comment:
The point is that this method, MIGHT return `int`, though will return `float` in most cases.